src/Traduora/TraduoraClient.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Refit;
+using Traduora.Client.Api;
+
+namespace Traduora.Client
+{
+	public class TraduoraClient
+    {
+        private readonly HttpClient _httpClient;
+        private const string BaseUrl = "https://traduora.azurewebsites.net/api/v1";
+        private const string DefaultFormat = "jsonnested";
+
+        public TraduoraClient(bool logging = false)
+        {
+            _httpClient = logging ? new HttpClient(new HttpLoggingHandler()) : new HttpClient();
```

Comment:
Use http client factory.

Reply:
We need to talk more about this one. Maybe we can do this in the next PR